<?xml version="1.0"?>
<launch>

<node pkg="gspn_framework_package" ns="robot_1" type="gspn_execution_node.py" name="executor_1" output="screen">
  <param name="user_robot_id" value="1" />
  <param name="user_current_place" value="p1" />
</node>

</launch>
